test/test.js

```diff
@@ -554,4 +562,440 @@ describe('parser (exif data)', () => {
+				assert.equal(exif.PlanarConfiguration, 1)
+				assert.equal(exif.ResolutionUnit, 1)
+				assert.deepEqual(exif.PageNumber, [ 0, 1 ])
+				assert.equal(exif.Software, 'V06.02.20')
+				assert.equal(exif.SampleFormat, 1)
+				assert.equal(exif.CameraSerialNumber, '297394')
+				assert.equal(exif.FNumber, 1.25)
+				assert.deepEqual(exif.DateTimeOriginal, new Date('2019-05-24T13:24:01.000Z'))
+				assert.equal(exif.FocalLength, 13)
+				assert.equal(exif.ImageNumber, 0)
+				assert.equal(exif.SubSecTimeOriginal, '52')
+				assert.equal(exif.FocalPlaneXResolution, 3.84)
+				assert.equal(exif.FocalPlaneYResolution, 3.072)
+				assert.equal(exif.FocalPlaneResolutionUnit, 4)
+				assert.equal(exif.SensingMethod, 15)
+				assert.equal(exif.GPSLatitudeRef, 'N')
+				assert.deepEqual(exif.GPSLatitude, [ 39, 55, 3.493 ])
+				assert.equal(exif.GPSLongitudeRef, 'W')
+				assert.deepEqual(exif.GPSLongitude, [ 2, 49, 22.714 ])
+				assert.equal(exif.GPSAltitudeRef, 0)
+				assert.equal(exif.GPSAltitude, 881.609)
+				assert.equal(exif.GPSMapDatum, 'WGS-84')
+				assert.equal(exif.latitude, 39.91763694444444)
+				assert.equal(exif.longitude, -2.8229761111111107)
+
+				// XMP
+				assert.exists(exif.xmp)
+			})
+
+		})
+
+		describe('FLIR DUO PRO 640 R 13mm', () => {
+
+			it('RGB', async () => {
+				const exif = await parse(buffers['FLIR_DUO_PRO_640_R_13mm.JPG'], options)
+				assert.exists(exif, `exif doesn't exist`)
+				assert.equal(exif.Make, 'FLIR')
+				assert.equal(exif.Model, 'Duo Pro R')
+				assert.equal(exif.XResolution, 72)
+				assert.equal(exif.YResolution, 72)
+				assert.equal(exif.ResolutionUnit, 2)
+				assert.equal(exif.Software, 'V01.02.05')
+				assert.equal(exif.YCbCrPositioning, 1)
+				assert.equal(exif.FNumber, 1.8)
+				assert.equal(exif.FocalLength, 8)
+				assert.equal(exif.FocalPlaneXResolution, 10.88)
+				assert.equal(exif.FocalPlaneYResolution, 8.704)
+				assert.equal(exif.FocalPlaneResolutionUnit, 4)
+				assert.equal(exif.ExifVersion, '0210')
+				assert.equal(exif.ComponentsConfiguration, 'Y, Cb, Cr, -')
+				assert.deepEqual(exif.SubjectArea, [ 2000, 1500, 4000, 3000 ])
+				assert.equal(exif.FlashpixVersion, '0100')
+				assert.equal(exif.ColorSpace, 1)
+				assert.equal(exif.PixelXDimension, 4000)
+				assert.equal(exif.PixelYDimension, 3000)
+				assert.equal(exif.GPSVersionID, '3.2.0.0')
+				assert.equal(exif.GPSLatitudeRef, 'S')
+				assert.deepEqual(exif.GPSLatitude, [ 0, 0, 0 ])
+				assert.equal(exif.GPSLongitudeRef, 'W')
+				assert.deepEqual(exif.GPSLongitude, [ 0, 0, 0 ])
+				assert.equal(exif.GPSAltitudeRef, 0)
+				assert.equal(exif.GPSAltitude, 0)
+				assert.equal(exif.GPSTimeStamp, '12:1:23')
+				assert.equal(exif.GPSSpeedRef, 'K')
+				assert.equal(exif.GPSSpeed, 0)
+				assert.equal(exif.GPSTrackRef, 'T')
+				assert.equal(exif.GPSTrack, 0)
+				assert.equal(exif.latitude, 0)
+				assert.equal(exif.longitude, 0)
+
+				// XMP
+                // TODO: Fix XMP
```

Comment:
The XMP of the FLIR DUO PRO has a weird format, causing it to be deleted entirely by the post processing, something to research further if we ever need the XMP.